scopes/component/ui/version-dropdown/version-dropdown.module.scss

```diff
@@ -1,4 +1,7 @@
+@import '~@teambit/ui-foundation.ui.constants.z-indexes/z-indexes.module.scss';
```

Comment:
can be removed from this PR

Will be fixed in #6714 
https://github.com/teambit/bit/pull/6714/files#diff-3fa370f6626e7ecad761bf8adfafad3e23458aaf5146c8bcb382440f7379b7e8